docs/business.yaml

```diff
@@ -1902,6 +2491,65 @@ components:
                 orderDate: '2019-08-24T14:15:22Z'
                 effectOfOrder: The Company will cease operations
                 orderDetails: The Company is ordered to dissolve
+    Amalgamation_application:
+      type: object
+      title: Amalgamation Application Filing
+      description: 'Amalgamation_application'
+      properties:
+        amalgamationApplication:
+          type: object
+          required:
+            - type
+            - amalgamatingBusinesses
+            - nameRequest
+            - parties
+            - contactPoint
+            - courtApproval
+          properties:
+            type:
+              type: string
+              title: The type of Amalgamation
+              enum:
+              - regular
+              - vertical
+              - horizontal
+            amalgamatingBusinesses:
+              type: array
+              items:
+                required:
+                - role
+                - identifier
+                properties:
+                  role:
+                    type: string
+                    enum:
+                    - amalgamating
+                    - holding
+                    - primary
+                  identifier:
+                    type: string
+                  foreignJurisdiction:
+                    $ref: '#/components/schemas/Foreign_jurisdiction'
+                  legalName:
+                    type: string
+            nameRequest:
+              title: The resulting business name and type
+              $ref: '#/components/schemas/Name_request'
+            nameTranslations:
+              $ref: '#/components/schemas/Name_translations'
+            contactPoint:
+              $ref: '#/components/schemas/Contact_point'
+            parties:
+              type: array
+              items:
+                $ref: '#/components/schemas/party'
+            courtOrder:
+              $ref: '#/components/schemas/Court_order'
+            courtApproval:
+              title: Amalgamation statement
+              type: boolean
```

Comment:
Consider adding these items to the amalgamation schema? Since they are in the Business Schemas. ( https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/amalgamation_application.json#L108C1-L121C30 )

Reply:
added

Reply:
not sure, do we also need this part in the new added `offices`? 
![image](https://github.com/user-attachments/assets/c14e710d-b86c-4ac9-a102-070ea6d3cf80)

Reply:
I think it's not always required only for Regular

Reply:
added